pennylane/qnode_new/jacobian.py

```diff
@@ -335,3 +337,53 @@ def _pd_analytic_var(self, idx, args, kwargs, **options):
             array[float]: partial derivative of the node
         """
         raise NotImplementedError
+
+    def to_torch(self):
```

Comment:
One thing I noticed when digging through QNodes is that somewhere inside, a torch QNode still sees itself as an autograd QNode (e.g., when you print the QNode, it says "Interface=NumPy/autograd" and there is still some autograd wrapping hanging around the internal functions). 

Hopefully this has gone away with the new QNode refactor

Reply:
Oops, this is a bug! Is this the case with the old QNode? Will mark it to fix in the new QNodes (if it still occurs).